src/test/java/uk/gov/justice/digital/hmpps/vsip/services/clients/PrisonVisitsTestingHelperClient.java

```diff
@@ -199,6 +202,96 @@ public void deleteApplication(String applicationReference) {
     }
 
 
+    public String createSessionTemplate(
+            String prisonCode,
+            LocalDateTime sessionStartDateTime,
+            Integer weeklyFrequency,
+            Integer closedCapacity,
+            Integer openCapacity,
+            String locationLevels,
+            String incentive,
+            String category,
+            Boolean disableAllOtherSessionsForSlotAndPrison
+    ) {
+        final var uri = "/test/prison/"+prisonCode+"/add/template";
```

Comment:
Small suggestion to edit the URL to the following:

final var uri = "/test/prison/"+prisonCode+"/template/add";